bedevere/prtype.py

```diff
@@ -4,18 +4,19 @@
 
 from . import util
 
-LABEL_PREFIX = "type"
+TYPE_LABEL_PREFIX = "type"
 
 
 @enum.unique
 class Category(enum.Enum):
     """Category of Pull Request."""
-    bugfix = f"{LABEL_PREFIX}-bugfix"
-    documentation = f"{LABEL_PREFIX}-documentation"
-    enhancement = f"{LABEL_PREFIX}-enhancement"
-    performance = f"{LABEL_PREFIX}-performance"
-    security = f"{LABEL_PREFIX}-security"
-    tests = f"{LABEL_PREFIX}-tests"
+
+    type_bug = f"{TYPE_LABEL_PREFIX}-bug"
+    documentation = "docs"
+    type_feature = f"{TYPE_LABEL_PREFIX}-feature"
+    performance = "performance"
+    type_security = f"{TYPE_LABEL_PREFIX}-security"
+    tests = "tests"
```

Comment:
`type-crash` is missing from the list, but it probably doesn't matter.

Reply:
Yeah I think bedevere really only cares about the `docs` and `tests` types at this moment. It doesn't do anything for the other type- labels.